web/packages/teleport/src/Discover/Shared/SelectCreatable/SelectCreatable.tsx

```diff
@@ -18,33 +18,39 @@ import React from 'react';
 import { useTheme } from 'styled-components';
 import ReactSelectCreatable from 'react-select/creatable';
 
+import { StyledSelect } from 'shared/components/Select/Select';
+
 const styles = theme => ({
   multiValue: (base, state) => {
-    return state.data.isFixed ? { ...base, backgroundColor: 'gray' } : base;
+    return state.data.isFixed
+      ? {
+          ...base,
+          backgroundColor: `${theme.colors.grey['600']} !important`,
```

Comment:
Are we able to avoid adding `colors.grey` in the first place, like replace it here with another color? This seems to be the only place which uses it. 😶‍🌫️

Reply:
thanks for the callout, i misunderstand how this all worked. i am closing this PR to address this (specific to select creatable) later, the other hard coded removal (replacing with existing theme colors) is addressed here instead: https://github.com/gravitational/teleport/pull/25438